src/main/java/com/google/sps/data/MapsRequest.java

```diff
@@ -25,19 +25,49 @@
 import com.google.maps.model.GeocodingResult;
 import com.google.maps.model.TravelMode;
 import com.google.maps.model.LatLng;
+import com.google.maps.model.PlacesSearchResponse;
+import com.google.maps.model.PlacesSearchResult;
+import com.google.maps.PlacesApi;
+import com.google.maps.model.PlaceType;
 import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;  
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
 
 /**
  * Class that creates a GeoApiContext variable and handles requests to maps API.
  */
 public class MapsRequest {
   private static GeoApiContext geoApiContext = null;
-  public static GeoApiContext getGeoApiContext() {
-    // TODO[ak47na]: provide Maps API key
+
+  private static String getApiKeyFromFile() throws IOException {
+    File apiKeyFile;
+    try {
+      apiKeyFile = new File(MapsRequest.class.getResource("/apiKey.json").toURI());
+    } catch (URISyntaxException e) {
+      e.printStackTrace();
+      return null;
```

Comment:
Optional: Silently swallowing exceptions can be problematic.  If this is "ok" here, it is often useful to leave a comment to explain why we're ignoring the exception. Otherwise it is a good idea to leave a `//TODO` to remind us to come back and fix it.

Reply:
Right, I changed `getApiKeyFromFile` to throw `ParseException` and the exception is now handled in `getGeoApiContext`.